eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/EventualCassandra.scala

```diff
@@ -52,8 +55,8 @@ object EventualCassandra {
   ](
     schemaConfig: SchemaConfig,
     origin: Option[Origin],
-    metrics: Option[EventualJournal.Metrics[F]]
-  ): F[EventualJournal[F]] = {
+    metrics: Option[EventualJournal.Metrics[F]],
+  )(implicit c: ConsistencyConfig.Read, w: ConsistencyConfig.Write): F[EventualJournal[F]] = {
```

Comment:
here I propose to pass single `consistencyConfig: ConsistencyConfig` covering both cases

Reply:
addressed